src/design/themes.stories.mdx

```diff
@@ -23,15 +13,17 @@ Themes may be applied to a containing element to change that content's appearanc
 The default theme, applied to the document `:root` by default.
 
 <Canvas>
-  <Story name="Light">{demoStory.bind({})}</Story>
+  <Story name="Light" args={{ theme: 'light' }}>
```

Comment:
Should this be `light` or `t-light`?

Reply:
In this case this is me telling the demo which theme is applied so it can modify its text and stuff accordingly, so even though it looks really close to the parameters it's not actually the same thing. I'm hoping this will only happen in the _theme_ demos.